auth-web/src/models/business.ts

```diff
@@ -146,6 +147,6 @@ export interface LearBusiness {
     identifier: string
     legalName: string
     legalType: string
-    alternateNames: AlternateNames[]
+    alternateNames: AlternateNameIF[]
```

Comment:
So this properties (array) is required here but optional in AffiliationResponse interface?